app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split(" ")
+    if command_parts[0] != "mv" or len(command_parts) != 3:
```

Comment:
Change order because if len(command_parts) is 0 it will give exception
```suggestion
    if len(command_parts) != 3 or command_parts[0] != "mv":
```